### PR TITLE
feat(osd-sts-ack): auto-add ack [OSD-8884]

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

File: deploy/osd-sts-ack/4.9/config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major/minor
+    operator: In
+    values: ["4.8"]
+  - key: api.openshift.com/sts-4.9-upgradeable
```
Comment (cblecker) [inline deploy/osd-sts-ack/4.9/config.yaml:7]: Are we changing this key every new version?

Comment (georgettica) [reply deploy/osd-sts-ack/4.9/config.yaml:7]: No, as the migration is only 4.8->4.9 AFAICT

Comment (cblecker) [reply deploy/osd-sts-ack/4.9/config.yaml:7]: No, there will potentially be STS changes with each minor version

Comment (georgettica) [reply deploy/osd-sts-ack/4.9/config.yaml]: solved based on your suggestion

**Comment by georgettica** [comment]
json patch is incorrect, will create a PR to fix this 
```
failed to apply patch 0: Invalid JSON Patch
```

**Pull request merged**